types/vivus/index.d.ts

```diff
@@ -1,6 +1,6 @@
-// Type definitions for Vivus 0.4.0
+// Type definitions for Vivus 0.4.1
```

Comment:
Looks like our lint rules are disabled for this package. Can you enable the lints in our `tslint.json` and fix up the version string to only be a `MAJOR.MINOR` version (e.g. `4.7`), as opposed to including the patch or containing a leading `v` (e.g. `v1.2.3`).